Dockerfile

```diff
@@ -82,5 +82,7 @@ COPY --from=build /app/_build/prod/rel/livebook /app
 # Make release files available to any user, in case someone
 # runs the container with `--user`
 RUN chmod -R go=u /app
+# Ditto allow Mix and projects to write to the user HOME
```

Comment:
We already set `777` on `$HOME` above, so it's writable by anyone, but the `.mix` dir generated with `mix local.hex --force` still has build-time-user permissions, so I would reword like this:

```suggestion
# Also, make all home files available (specifically .mix)
```